src/ui/home/components/identity.tsx

```diff
@@ -27,44 +33,98 @@ const styles = StyleSheet.create({
     marginBottom: "15%"
   },
   icon: {
-    marginLeft: "75%",
-    marginBottom: "15%"
+    margin: "20%"
+  },
+  iconContainer: {
+    height: 55,
+    width: 55,
+    borderRadius: 35,
+    backgroundColor: '#942f51',
```

Comment:
Should we try to incorporate some of these styling conditions into the new Jolocom-Theme?

Reply:
Refereced backgroundColor from Jolocom-Theme.